### ISSUE TITLE
Implement bool filter with Rules for Opening/Closing positions by fuzzy Risk/Reach levels

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#117]
This is the author's technique, proposed by [Timur Gilmullin](https://www.linkedin.com/in/tgilmullin), based on fuzzy scales for measuring the levels of fuzzy risk and reachable. The following diagram explains it:

![005-Open-Close-Rules-Matrix](https://user-images.githubusercontent.com/7660870/211887277-1de80553-4ed7-42b2-bea7-d603f6f10c67.png)

Here in table `T` mean `True`, `F` mean `False`. 1st position is for Opening rules, 2nd position is for Closing rules.

See also, about universal fuzzy scale:
- (EN) https://github.com/devopshq/FuzzyRoutines#Chapter_2_4
- (RU) https://math-n-algo.blogspot.com/2014/08/FuzzyClassificator.html#chapter_3

```
R i s k
e        Min  Low  Med  High Max
a
c   Min  F/F  F/F  F/T  F/T  F/T
h
a   Low  F/F  F/F  F/T  F/T  F/T
b
i   Med  T/F  T/F  T/F  F/T  F/T
l
i   High T/F  T/F  T/F  F/F  F/F
t
y   Max  T/F  T/F  T/F  F/F  F/F
```

**Comment by Tim55667757** [comment]
Implemented since v1.6.***

**Comment by Tim55667757** [comment]
Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/259728758
Artifact for testing in PyPI: https://pypi.org/project/tksbrokerapi/1.6.dev138/
